sdks/python/tests/library_integration/langchain/test_langchain.py

```diff
@@ -155,7 +156,9 @@ def test_langchain__openai_llm_is_used__token_usage_is_logged__happyflow(
     synopsis_chain = prompt_template | llm
     test_prompts = {"title": "Documentary about Bigfoot in Paris"}
 
-    callback = OpikTracer(tags=["tag1", "tag2"], metadata={"a": "b"})
+    callback = OpikTracer(
+        tags=["tag1", "tag2"], metadata={"a": "b"}, project_name="e2e-langchain"
```

Comment:
`project_name` is not checked, we should either remove it (which is totally fine since the test is not about it), or add to expected traces/spans.